src/test/java/plugins/LogParserTest.java

```diff
@@ -15,11 +15,16 @@
 import java.util.Map;
 
 import static org.hamcrest.MatcherAssert.*;
-import static org.jenkinsci.test.acceptance.Matchers.*;
+import static org.hamcrest.Matchers.*;
+import org.jenkinsci.test.acceptance.plugins.logparser.LogParserOutputPage;
+import static org.junit.Assert.assertEquals;
 
 @WithPlugins("log-parser")
 public class LogParserTest extends AbstractJUnitTest {
 
+    // XPath to the summary on the build page
```

Comment:
// comments  tend to get outdated

So better call the constant SUMMARY_XPATH then no comment is required